cypress/lib/Player.js

```diff
@@ -1,16 +1,30 @@
 import { Game } from "./Game";
 
-export const Rebuttals = {
-  RIGHT: "Right",
-  LEFT: "Left",
-};
-
 export const Player = {
+  setsGuessForPoints(points) {
+    Game.getGuessForPoints(points).then((guessAngle) => {
+      Player.setsGuess(guessAngle);
+    });
+  },
   setsGuess(guess) {
     Game.enablePlayerView();
     cy.get('[data-cy="game_input_guess"]').clear().type(guess);
     cy.get('[data-cy="game_btn_submit_guess"]').click();
   },
+  choosesRebuttalWithCorrectness(correct) {
+    let rebuttalFunction = correct
+      ? Game.getCorrectRebuttal.bind(Game)
+      : Game.getIncorrectRebuttal.bind(Game);
```

Comment:
can you try removing the `.bind()` calls here? they shouldn't be necessary since `Game` doesn't use `this` anymore